commands/version/index.js

```diff
@@ -187,10 +187,14 @@ class VersionCommand extends Command {
       );
     }
 
+    const opts = Object.assign({}, this.execOpts);
+    if (!this.project.isIndependent() && this.tagPrefix) {
+      opts.match = `${this.tagPrefix}*`;
```

Comment:
It's probably better to use `**` instead to allow for tags that contain forward slashes as a single star will not match those.
```suggestion
      opts.match = `${this.tagPrefix}**`;
```